google-cloud-datastore/src/test/java/com/google/cloud/datastore/DatastoreHelperTest.java

```diff
@@ -159,72 +153,4 @@ public void testFetchWithTransaction() throws Exception {
     assertEquals(entity2, values.get(1));
     verify(transaction);
   }
-
```

Comment:
Why was this removed?

Reply:
After I cleaned up the surface, I got rid of the DatastoreHelper "default method", hence deleting the tests. The new impl logics are tested in DatastoreTest and ITs.